analyzers/dataframe/src/myUtils.cc

```diff
@@ -2051,7 +2089,7 @@ ROOT::VecOps::RVec<float> get_DVertex_thrusthemis_angle(ROOT::VecOps::RVec<Verte
   ROOT::VecOps::RVec<float> result;
 
   for (auto &p:vertex){
-    if (p.vertex.primary>0)continue;
+    if (isPrimaryOrSecondaryVtx(p.vertex))continue;
```

Comment:
This is one of the places where I am not sure whether we only want the primary vertex or primary and secondary.

Reply:
Analyses that are using this are:
- flavour/B2Kstee
- flavour/Bd2KstNuNu
- flavour/Bc2TauNu
- flavour/BuBc2TauNu
- flavour/Bs2PhiNuNu

Reply:
Hi @tmadlener for the b2snunu analyses, Bd2KstNuNu and Bs2PhiNuNu,  we are trying to reconstruct the Kst or phi so don't want to be using the PV since that's the B decay vertex.

Reply:
Right, so this should actually only cover secondary vertices? Or is the current check OK that will be true for primary and secondary vertices? The latter would preserve existing behavior, I think.

Reply:
We just want it to skip primary vertices, so I think we only want it to do `continue` if `isPrimaryVtx(p.vertex)` is `True` right?

Reply:
That make sense. Thanks for the clarification. I will change this accordingly.